spdx/parsers/jsonyamlxmlbuilders.py

```diff
@@ -205,6 +205,22 @@ def set_file_type(self, doc, type_value):
 
         return super(FileBuilder, self).set_file_type(doc, f"namespace#fileType_{type_value.lower()}")
 
+    def set_file_copyright(self, doc, text):
+        """
+        Raise OrderError if no package or file defined.
```

Comment:
Luckily, packages are out of the equation now :)

Reply:
deleted